src/app/cli/src/coda_shared_prefix_multiproposer_test.ml

```diff
@@ -5,18 +5,21 @@ open Coda_main
 
 let name = "coda-shared-prefix-multiproposer-test"
 
-let main () =
+let main n () =
   let logger = Logger.create () in
-  let n = 2 in
   let snark_work_public_keys i = None in
   let%bind testnet =
     Coda_worker_testnet.test logger n Option.some snark_work_public_keys
       Protocols.Coda_pow.Work_selection.Seq
   in
-  let%bind () = after (Time.Span.of_sec 30.) in
+  let%bind () = after (Time.Span.of_min 10.) in
```

Comment:
In 10 minutes, what are the things we are expecting the test to finish? 
Like how many blocks are we expecting each proposer to have?

Reply:
Concretely we want to be able to have this test have caught the "empty epoch bug", which means it needs to run for at least 2 full epochs after the first ledger proof is emitted. I haven't done all the math to figure how long that is yet -- `10m` is long enough that I usually see the flakes cause it to fail. anything much shorter tends to sometimes hide the errors.